licenses/CAMUNDA-LICENSE-1.0.txt

```diff
@@ -0,0 +1,65 @@
+refrain from Using the Software. If You receive the Software in original or
+modified form from a third party, the terms and conditions outlined in this
+License apply to Your Use of that Software. You should have received a copy
+of this License in this case.
+ 
+Copyright License
+Subject to the terms and conditions of this License, the Licensor hereby grants
+You the non-exclusive, royalty-free, worldwide, non-sublicensable, non-transferable
+right to Use the Software in any way or manner that would otherwise infringe the
+Licensor’s copyright as long and insofar as You Use the Software only and limited
+to the Use in or for the purpose of Using the Software in Non-Production Environment.  
+Each time you distribute or make otherwise publicly available the Software or
+Derivative Works thereof, the recipient automatically receives a license from
+the original Licensor to the respective Software or Derivative Works thereof
+under the terms of this License.
+
+Conditions and Restrictions
+All Use of the Software is explicitly made subject to the following conditions:
+  * You may not move, change, disable, or circumvent the license key functionality
+    in the Software, and You may not remove or obscure any functionality in the
+    Software that is protected by the license key.
+  * If You distribute or make available the Software or any modification or Derivative
+    Works thereof (including compiled versions), You must conspicuously display and
+    attach this License on each original or modified copy of the Software and enable
+    the recipient to obtain the source code if You have distributed a compiled version
+
+
+Patent License
+Patent and trademark rights are not licensed under this Public License.
+
+
+No Liability
+EXCEPT FOR DAMAGES CAUSED BY INTENT OR FRAUDULENTLY CONCEALED DEFECTS, AND EXCEPT FOR
+DAMAGES RESULTING FROM BREACH OF ANY WARRANTY OR GUARANTEE EXPRESSLY GIVEN BY LICENSOR
+IN THIS LICENCE, IN NO EVENT WILL LICENSOR BE LIABLE TO YOU ON ANY LEGAL THEORY FOR ANY
+DAMAGES ARISING OUT OF THIS LICENSE OR THE USE OF THE WORK. ANY MANDATORY STATUTORY
+LIABILITY UNDER APPLICABLE LAW REMAINS UNAFFECTED.
+ 
+No Warranty
+EXCEPT AS EXPRESSLY STATED IN THIS LICENSE OR REQUIRED BY APPLICABLE LAW, THE WORKS ARE
+PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND INCLUDING WITHOUT LIMITATION,
+ANY WARRANTIES REGARDING THE CONTENTS, ACCURACY, OR FITNESS FOR A PARTICULAR PURPOSE.
+
+
+Definitions
+You refer to the individual or entity agreeing to these terms.
+Use means any action concerning the Software that, without permission, would make Youliable
+for infringement under applicable copyright law. Use within the meaning of this License
+includes, but is not limited to, copying, distribution (with or without modification),
+making available to the public, and modifying the Software.
+
+Non-Production Environment means a setting in which the Software is used for development, stagging,
```

Comment:
That's looks like a typo, should be "staging", right?

Reply:
I just copied & pasted it 😄 But yes, I think so, I will adjust it.